core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20250211_add_workspace_spend_report_table.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
```

Comment:
this is using `dbchangelog-3.4.xsd`, i.e. only supports Liquibase 3.4 syntax. We should be able to go all the way to [dbchangelog-4.31.xsd](https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.31.xsd); the [most recent changeset](https://github.com/broadinstitute/rawls/blob/f246f1f651231c873329b1b7d15c68192acd13e0/core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20241218_alter_audit_workflow_status_trigger_frequency.xml#L5) uses 4.30.

Reply:
Thanks for catching that! I copied one of the older changesets so I'll update that